control/statefbk.py

```diff
@@ -373,9 +374,9 @@ def lqr(*args, **keywords):
 
     Returns
     -------
-    K: 2D array
+    K: 2D array (or matrix)
```

Comment:
numpydoc:

```suggestion
    K : 2D array (or matrix)
```
Same for all the other functions in this file, which are a mix of numpydoc and non-numpydoc right now.